modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/ExtensionDefinitionParser.java

```diff
@@ -211,6 +214,29 @@ protected ExtensionDefinitionParser(Builder baseDefinitionBuilder, DslSyntaxReso
    */
   protected abstract Builder doParse(Builder definitionBuilder) throws ConfigurationException;
 
+  protected void parseNestedComponents(List<? extends NestableElementModel> nestedComponents) {
```

Comment:
seems like this whole method is a left over

Reply:
nope, used in OperationDefinitionParser